docs/tutorials/data/hess.ipynb

```diff
@@ -12,7 +12,17 @@
     "\n",
```

Comment:
<p>Do we need this? Would it really work like this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/3350/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>